Makefile

```
@@ -8,4 +8,7 @@ apply:
 destroy:
 	aws-vault exec moj-nac-shared-services --duration=2h -- terraform destroy
 
+perf-test-setup:
+	sh ./scripts/perf_test_setup.sh                                               
+
 .PHONY: init apply destroy
```

```suggestion
.PHONY: init apply destroy perf-test-setup
```

I wonder if we need to add the `aws-vault...` command here too(?).

---

That makes sense. We could also add an argument to the `make` command for each environment, but it's not necessary.